src/pages/Staking/index.tsx

```diff
@@ -534,29 +537,30 @@ const Staking = (): JSX.Element => {
   };
   const stakedAmount = getStakedAmount();
 
-  React.useEffect(() => {
-    if (!governanceTokenBalance || !stakedAmount) return;
+  const availableBalance = React.useMemo(() => {
```

Comment:
@nud3l 
With `useMemo`, the code could be more simplified and it's the use case of `useMemo` rather than `useEffect`.
As the function passed to `useMemo` is executed during rendering unlike the one passed to `useEffect`, we do not have to worry about the default value as well.